cpp/ql/src/semmle/code/cpp/exprs/Assignment.qll

```diff
@@ -4,11 +4,13 @@ import semmle.code.cpp.exprs.BitwiseOperation
 
 /**
  * A non-overloaded binary assignment operation, including `=`, `+=`, `&=`,
- * etc. A C++ overloaded operation looks syntactically identical but is instead
+ * etc. A C++ overloaded `operator` looks syntactically identical but is instead
  * a `FunctionCall`.
+ *
+ * This is an abstract root QL class for all (non-overloaded) assignments.
  */
 abstract class Assignment extends Operation {
-  /** Gets the lvalue of this assignment. */
+  /** Gets the *lvalue* of this assignment. */
```

Comment:
Can anyone remember whether we decided to keep this formatting (`*lvalue*`)?

Reply:
We decided against formatting code with asterisks, but I think in this case it's meant to denote italics because *lvalue* is not an English word. Allows writing italics with single asterisks (bold is double asterisks), but it's more consistent if we stick to underscores, changing this to `_lvalue_`.

Reply:
I've done a search-and-replace to change `*lvalue*` into `_lvalue_` (and similarly for `*rvalue*` etc).